ocs_ci/utility/utils.py

```diff
@@ -3334,7 +3205,15 @@ def check_if_executable_in_path(exec_name):
     return which(exec_name) is not None
 
 
-def upload_file(server, localpath, remotepath, user=None, password=None, key_file=None):
+def upload_file(
+    server,
+    localpath,
+    remotepath,
+    user=None,
+    password=None,
+    key_file=None,
+    jump_host_connection=None,
```

Comment:
Please document this argument in the docstring (looks like, we are missing there also the `password` and `key_file` parameters, so please if you can include them too).

If I understand it correctly, the `jump_host_connection` is "an existing ssh connection" which is used for the sftp transfer, right? In that case, I would propose to change the name to something like `ssh_connection` or something like that (because it actually could be used for any connection, not only through jump host).